### ISSUE TITLE
Exempt admins from /tax, make /nos free

### ISSUE THREAD

**Issue by ghost** [LVPlayground/playground#92]
When the player is in the 30+ region, it can be hard to follow the chat with player messages, admin notices, PMs, etc. I'm trying to identify areas of improvement to reduce the amount of unnecessary information presented to admins. We recently removed vehicle export messages, and, I'd also like it if we can exempt admins from tax, therefore removing the **You have paid $xxx taxes.** messages.

Furthermore, given that most functions are already free for crew, such as /tow, /taxi, etc -- can we duplicate this behavior for /nos? 

This is very low priority, marking it as such.

**Comment by GodeyLVP** [comment]
Sorry for this spam!.
I'm new to GitHub, I'll make the original fix.
